app/views/errors/deleted_pid.html.erb

```diff
@@ -0,0 +1,21 @@
+<%#
+Copyright 2011-2017, The Trustees of Indiana University and Northwestern
+  University.  Licensed under the Apache License, Version 2.0 (the "License");
+  you may not use this file except in compliance with the License.
+
+You may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software distributed
+  under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR
+  CONDITIONS OF ANY KIND, either express or implied. See the License for the
+  specific language governing permissions and limitations under the License.
+---  END LICENSE_HEADER BLOCK  ---
+%>
+<div id="main_content_container" class="container-fluid">
+  <div class="content" class="col-md-8">
+    <h2>Item deleted</h2>
+    <p>The item you requested has been deleted from the system. If this is not expected, contact your support person.</p>
```

Comment:
@cjcolvar Can the second sentence read "If this is not expected, contact your support staff."? I think most institutions including us have a support queue or email rather than a specific person.